src/widgets/WidgetMainSmall.vue

```diff
@@ -177,6 +178,11 @@ Indicative monthly payment is a minimum monthly repayment (MMP) of the greater o
 ^ Indicative repayments (Transaction amount divided by Interest Free Period) are an estimate only, which excl $99 Annual Fee, and assumes no additional purchases, cash advances or other fees or charges. Interest Free Period available when indicative monthly repayments are made by each statement period due date, resulting in full repayment of purchase amount within the Interest Free Period. `,
     }
   },
+  beforeCreate() {
+    fetchData('/widget', {
+      method: 'POST',
+    })
+  },
```

Comment:
Nice. I wonder if we'll have to do this as a callback of `this.$nextTick(() => {` - could be good to check with JP and Anton. In addition, I think we should remove the data fetching for now and only put it back in once we start using the data

Reply:
Hi @antoninroam and @jpyepez-roam , before I remove the data fetching (for now), do you guys have any recommendations on how to fetch API data in the widget (nextTick, beforeCreate, etc)?